examples/angular/src/app/components/content/content.component.ts

```diff
@@ -1,13 +1,16 @@
 import { Component, Input, OnDestroy, OnInit } from "@angular/core";
 import { providers, utils } from "near-api-js";
-import { AccountView, CodeResult } from "near-api-js/lib/providers/provider";
-import { WalletSelector, AccountState } from "@near-wallet-selector/core";
-
-import { Message } from "../../interfaces/message";
-import { Sumbitted } from "../form/form.component";
-import { Account } from "../../interfaces/account";
+import type {
+  AccountView,
+  CodeResult,
+} from "near-api-js/lib/providers/provider";
+import type { WalletSelector, AccountState } from "@near-wallet-selector/core";
+
+import type { Message } from "../../interfaces/message";
+import type { Submitted } from "../form/form.component";
+import type { Account } from "../../interfaces/account";
 import { distinctUntilChanged, map, Subscription } from "rxjs";
```

Comment:
Is `Subscription` a type here?

Reply:
I wasn't sure for this since this how the subscription class is declared:

```ts
export class Subscription implements SubscriptionLike {}
```